README.md

```diff
@@ -51,7 +117,7 @@ Additionally, it will compute a version string, using a hash function that takes
 Next, it will look for an existing instance of a `Workload CRD` for the given workload name:
 
 - If it finds the `Workload`, it will update its version according to the previously computed version string. In addition, it will include a reference to the ReplicaSet UID of the pod (i.e. the Pods owner), or the pod itself, if it does not have an owner.
-- If it does not find a workload instance, it will create one containing the previously computed version string. In addition, it will include a reference to the ReplicaSet UID of the pod (i.e. the Pods owner), or the pod itself, if it does not have an owner. 
+- If it does not find a workload instance, it will create one containing the previously computed version string. In addition, it will include a reference to the ReplicaSet UID of the pod (i.e. the Pods owner), or the pod itself, if it does not have an owner.
```

Comment:
Protip: split paragraph and sentences to multiple lines to make it more maintenable